src/main/scala/is/hail/variant/VariantDataset.scala

```diff
@@ -837,9 +837,9 @@ class VariantDatasetFunctions(private val vds: VariantSampleMatrix[Genotype]) ex
     vds.annotateSamples(result, signature, "sa.imputesex")
   }
 
-  def linreg(ySA: String, covSA: Array[String], root: String, minAC: Int, minAF: Double): VariantDataset = {
+  def linreg(ySA: String, covSA: Array[String], root: String, useDosages: Boolean, minAC: Int, minAF: Double): VariantDataset = {
```

Comment:
oh, there actually aren't default args here. Can you add all the default args that python has?